doc/tests.md

```diff
@@ -0,0 +1,68 @@
+# Travis CI
+
+Every pull request runs the full suite of functional tests, and unit tests on Travis CI. We test on Chrome stable and beta, on Firefox we test ESR, latest, and beta. See our `[.travis.yml](/.travis.yml)` for the configuration, as well as [`scripts/setup_travis.sh`](scripts/setup_travis.sh) and
+[`scripts/run_travis.sh`](scripts/run_travis.sh).
+
+We also run [ESLint](https://eslint.org) on Travis to test for JavaScript style and errors. See `.eslintrc.yml` and `.eslintignore` for specifics.
+
+# Unit tests
+
+To run the unit tests, click on the badger icon next to the URL bar to open the popup.
+Then in the popup, click on the gear icon (⚙) to open the options page.
+Your browser should navigate to an internal URL that starts with `chrome-extension://` or `moz-extension://` and ends with `/skin/options.html`.
+Replace `/skin/options.html` with `/tests/index.html` and hit enter.
+This will open the unit tests and run them.
+
+Unittests are located in `privacybadger/src/tests/tests`. The unit test dependencies are in `privacybadger/src/tests/lib`. Please add unittests whenever possible.
```

Comment:
Should remove the "privacybadger" part of the path from both paths. Also turn the paths into links?